app/assets/javascripts/report-a-problem-form.js

```diff
@@ -79,5 +80,11 @@
     this.$form.trigger('reportAProblemForm.success', data);
   };
 
+  ReportAProblemForm.prototype.trackEvent = function(action){
+    if (GOVUK.analytics && GOVUK.analytics.trackEvent) {
```

Comment:
Due to the way we load modules there's a chance ReportAProblemForm could be loaded without the Analytics module.